agent/topology_probes.go

```diff
@@ -47,6 +48,8 @@ import (
 // NewTopologyProbe creates a new topology probe
 func NewTopologyProbe(name string, ctx tp.Context, bundle *probe.Bundle) (probe.Handler, error) {
 	switch name {
+	case "blockdev":
+		return new(blockdev.ProbeHandler).Init(ctx, bundle)
```

Comment:
We recently merged a PR so that Skydive handles probes as external .so
Therefore, we now try to use the `module.NewProbe(ctx, bundle)` scheme. Could you please do this small refactoring ?